class-tgm-plugin-activation.php

```diff
@@ -1337,6 +1362,22 @@ protected function display_settings_errors() {
 		}
 
 		/**
+		 * Display admin notice if plugins have been force activated.
+		 *
+		 * @since 2.6.0
+		 */
+		public function display_forced_activation_notice() {
+			if ( $force_activated_plugins = get_transient( 'tgmpa_force_activated_plugins' ) ) {
+				?>
+				<div class="notice <?php echo esc_attr( $this->get_admin_notice_class() ); ?> is-dismissible">
```

Comment:
No need to go in and out of PHP several times. Just gather the variables and echo message out once you've collected the parts using comma's between arguments.

Also, while the transient should never be empty, an extra defensive coding check to verify this before building the message would not go amiss.